### ISSUE TITLE
[Feature Request]: Functional satellite imagery in maps

### ISSUE THREAD

**Issue by tito-abbo** [Symbiota/Symbiota#1295]
### Symbiota Portal Name

CCH2

### Is your feature/tool request related to a problem? Please describe.

I noticed that the leaflet maps have nonfunctional toggles for "topo" and "satellite" imagery. I am assuming that this is not a bug and that this feature is pending. While I almost always prefer nongoogle products and openstreets as not an exception, the access to satellite imagery was a significant advantage to the previous mapping solution. Implausible locations are often easier to notice in satellite imagery (e.g. on the side of a cliff), and I have also used satellite imagery to get a coarse impression of vegetation.

**Comment by MuchQuak** [comment]
@tito-abbo These should be functional and I am seeing them work for firefox, chrome, and safari on cch2. Could I trouble you with what browser, with the version preferably, you are using, screen shots of issue so I can try to replicate this issue?

**Comment by tito-abbo** [comment]
@MuchQuak I tried it on a different record and initially was unable to recreate the issue, so I tested a little more and figured out what was going on. For the first record I looked at I was zoomed in more than what was available for the satellite or topo maps. Thus, those toggles were grayed out. Perhaps a message could be added to clarify that one must zoom out to change modes or the toggle could zoom out automatically, but I apologize for not trying more things and just assuming there was an issue.
